deploy/examples/keycloakRealm.json

```diff
@@ -9,7 +9,7 @@
         "realm": "arealm",
         "displayName": "arealm",
         "enabled": true,
-        "eventsListeners": ["metrics-listener"],
```

Comment:
why remove the metrics listener from the example?

Reply:
oh, it kept generating an error that was a bit distracting, I can re-add it though

Reply:
done.